### ISSUE TITLE
Where is the code that allows you to zoom in?

### ISSUE THREAD

**Issue by roser234** [nycholassousa/ravendawn-client-datapack-decrypted#3]
const.lua and sentting don't seem to have anything there.

**Comment by nycholassousa** [comment]
Maybe it's in the "gameinterface.lua" file, I haven't tried doing anything related to zoom yet

**Comment by roser234** [comment]
> Tal vez esté en el archivo "gameinterface.lua". Todavía no he intentado hacer nada relacionado con el zoom.

Yes, there it is. but the screen does not extend, only black tiles appear, if you can do something, advise me.

**Comment by nycholassousa** [comment]
> > Tal vez esté en el archivo "gameinterface.lua". Todavía no he intentado hacer nada relacionado con el zoom.
> 
> Yes, there it is. but the screen does not extend, only black tiles appear, if you can do something, advise me.

What have you changed? Can you send the pastebin of the file?

**Comment by roser234** [comment]
> > > Tal vez esté en el archivo "gameinterface.lua". Todavía no he intentado hacer nada relacionado con el zoom.
> > 
> > 
> > Sí, ahí está. pero la pantalla no se extiende, solo aparecen mosaicos negros, si puedes hacer algo avisame.
> 
> ¿Qué has cambiado? ¿Puedes enviar el Pastebin del archivo?

intente tocar un poco uigamemap.lua y interface pero no lo consigo.
gameMap:setKeepAspectRatio(true)
gameMap:setVisibleDimension({
https://pastebin.com/raw/4Gp5XUf0
